compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -31,6 +31,13 @@ enum class DataFormat
   NHWC
 };
 
+/**
+ * @brief Set annotation for DataFormat (NCHW, NHWC)
+ *
+ * @note DataFormatAnnotation will live longer than this Pass (until the
+ *       annotated loco::Node is erased). So, do not use large data in the
```

Comment:
I've not seen where `annotated loco::Node is erased`...Can you share where the code do this?

Reply:
I think this question was answered in the offline talk, so I check this conversation resolved.